commands/zcount.md

```diff
@@ -16,6 +16,6 @@ Note: the command has a complexity of just O(log(N)) because it uses elements ra
 ZADD myzset 1 "one"
 ZADD myzset 2 "two"
 ZADD myzset 3 "three"
-ZCOUNT myzset -inf +inf
+ZCOUNT myzset -inf +inf  // -inf +inf : The meaning of this sentence is infinity.
```

Comment:
You can't use `//` as a comment in redis-cli, so this is an error if the example is pasted in redis-cli.

Please add this description of negative infinity and positive infinity somewhere else in this document, together with the description of min and max.

Reply:
As you advised in the review, I added the contents of 'inf' and `//` to another place without using it. Please check. Thank you.